### DISCUSSION TITLE
Dark mode usage

### DISCUSSION THREAD

**Discussion by is-sam** [easepick/easepick#77] [Q&A] [answered]
How to force dark mode? I installed `@easepick/amp-plugin` and set `darkMode` to `true` but it still doesn't work even though dark mode is enabled in my system settings.

**Comment by wakirin** [comment, marked as answer]
Check index page with enabled dark mode in your system: https://easepick.com
You should see this:
<img width="704" alt="Screen Shot 2022-06-10 at 22 42 15" src="https://user-images.githubusercontent.com/7056055/173112742-b5bf5085-8181-4252-85ad-34be6f009e68.png">

There is currently no option in config to force dark mode on, but you can do something like this: https://jsfiddle.net/u5tyvzd4/

**Reply by is-sam** [reply]
It does show in dark mode on the index page, but it doesn't in my app... I will just use the solution you proposed, thank you 👍

**Reply by wakirin** [reply]
Check which css file you are include in css option.
I found the docs contains wrong css file `https://cdn.jsdelivr.net/npm/@easepick/core@1.2.0/dist/index.css` in quick example (Amp plugin).
It should be `https://cdn.jsdelivr.net/npm/@easepick/bundle@1.2.0/dist/index.css`.
I will update the docs later.
